luigi/contrib/kubernetes.py

```diff
@@ -158,6 +158,13 @@ def max_retrials(self):
         """
         return self.kubernetes_config.max_retrials
 
+    @property
+    def backoff_limit(self):
+        """
+        Maximum number of retries before considering the job as failed.
+        """
+        return 6
```

Comment:
Is there any particular reason to have 6? 🙂

Reply:
6 is the default when not specified, so it seems like a sensible default here as well.

Reply:
feel like we should default to one here

Reply:
@flylo hm I guess we could, but I thought it would be best to stay consistent with the default behavior here, and we can override this in our internal wrapper to be 1.

Reply:
sounds reasonable!

Reply:
Maybe add a link to k8s doc where it says `6` as default value?